security/test_signing.py

```diff
@@ -33,6 +113,20 @@ def resolve_public_key(self, key_id: str):
             return load_pem_public_key(fh.read())
 
 
+class _TestCertificationResolver:
+    def __init__(self, pvt_key, cert_creator: _TestCertificationCreator):
+        self._pvt_key = pvt_key
+        self.cert_creator = cert_creator
+
+    def resolve_private_key(self, key_id: str):
```

Comment:
I think we can remove this. The private key should not be visible or accessible via the certificate.